src/exo/API_scheduling.py

```diff
@@ -1512,6 +1726,12 @@ def inline_window(proc, winstmt_cursor):
 
     args:
         winstmt_cursor  - cursor pointing to the WindowStmt to inline
+        rc              - (bool) whether to return relevant cursors
+
+    relevant cursors:
+        exprs           - cursors to all replaced window expressions
+        TODO: unsure of this one, since it's hard to distinguish
```

Comment:
This is one of the cases where it might be worth returning modified cursors because after the edit, there's no way to identify what was previously using the variable from the window statement.

Reply:
I support this

Reply:
Actually, this may be tricky...

1) The alternative would be to use an introspection feature to get all the window use-sites before doing this operation and then count on forwarding to preserve the identities of all those use-sites.

2) The use-sites are not a homogeneous type of cursor, since the window could be used as both an lvalue or an rvalue.  We need to think more carefully about what exactly would get returned here, and how it relates to inspection features...